.bingo/variables.env

```diff
@@ -0,0 +1,16 @@
+# Auto generated binary variables helper managed by https://github.com/bwplotka/bingo v0.2.3. DO NOT EDIT.
```

Comment:
It also creates these shell script variables that can be used. Not as magic as the Makefile ones.